definitions/ext-rclusterhost/golden_metrics.yml

```diff
@@ -0,0 +1,15 @@
+rclusterCpuUsage:
+  title: CPU usage (%)
+  query:
+    select: average(rcluster.container.cpu.percentused)*100
+    facet: rfc460Hostname
```

Comment:
As far as I understand this property is not stored as a tag so it will not exist.

That said since this property is the `synthesis.name` we should be able to facet by `entityName` and have the same result.

We can remove this line since that is the default

Reply:
Are you referring to the `facet: rfc460Hostname`? Would we need to define it as a tag in definition.yml to use the facet? Since it is the `entityName` it would be a duplicate anyway.

Reply:
Yep sorry I didn't explain myself correctly.

Here you should use `facet: entity.name` since as you said `rfc460Hostname` will be the entity name attribute by default.

And `entity.name` is the default value as you can see in [our docs](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/golden_metrics.md#query-options) so we can remove this line and it will apply correctly.

Reply:
Thanks for the clarification!